client/src/views/auth/column/index.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable prettier/prettier */
+import ColumnForm from './ColumnForm'
+import { selectCareer, setKeywords, clearKeywords } from '../../../slices/careerSlice'
```

Comment:
why do you import components in careerSlice?

Reply:
those r useful